### DISCUSSION TITLE
face enhancer is freezing

### DISCUSSION THREAD

**Discussion by jonxq** [hacksider/Deep-Live-Cam#653] [Q&A]
Hello, in general deepfake working great without freezing , but when I turn on face enhancer my fps is become terrible, can I somehow turn ON better quality only for face ( not for background )  and it will make my fps better because its make better quality only for face?
If yes, help how to do it

**Comment by hacksider** [comment]
Make the preview smaller, that will always improve the performance but don't expect too much with enhancer turned on.
